tests/terraform/checks/resource/oci/test_SecurityListIngressStateless.py

```diff
@@ -26,20 +26,13 @@ def test(self):
             "oci_core_security_list.fail2",
         }
 
-        skipped_resources = {
-            "oci_core_security_list.skipped",
-        }
-
         passed_check_resources = set([c.resource for c in report.passed_checks])
         failed_check_resources = set([c.resource for c in report.failed_checks])
-        skipped_check_resources = set([c.resource for c in report.skipped_checks])
 
         self.assertEqual(summary["passed"], 3)
         self.assertEqual(summary["failed"], 2)
-        self.assertEqual(summary["skipped"], 1)
```

Comment:
maybe it will be better to check for unknown in the summary ? 
or is it being omitted from it ?

Reply:
Yes, it is omitted @omryMen